modules/fakemeta/fakemeta_amxx.cpp

```diff
@@ -61,19 +65,26 @@ void OnAmxxAttach()
 		return;
 	}
 
-	void *address = nullptr;
-
-	if (!CommonConfig->GetAddress("g_pGameRules", &address) || !address)
+	if ((HasRegameDll = RegamedllApi_Init()))
 	{
-		MF_Log("get/set_gamerules_* natives have been disabled because g_pGameRules address could not be found. ");
-		return;
+		GameRulesRH = ReGameApi->GetGameRules();
```

Comment:
At this stage, GetGameRules will return nullptr, but we can register a hook [InstallGameRules](https://github.com/s1lentq/reapi/blob/adb34bb0ed46c1ccd21e0ba5ab8bfcaae9db8116/reapi/src/main.cpp#L93) to get the g_pGameRules pointer.